jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.vue

```diff
@@ -134,12 +134,15 @@
                  && (bg_selected === 'None' || bg_selected_validity.is_aperture)
                  && dev_subpixel_support">
         <v-row>
-          <v-switch
-            v-model="subpixel"
-            label="Subpixel"
-            hint="Extract spectrum using a subpixel aperture in place of the subset mask"
-            persistent-hint>
-          </v-switch>
+          <v-select
+            :menu-props="{ left: true }"
+            attach
+            :items="aperture_method_items"
```

Comment:
This should fix what @rosteen reported

```suggestion
            :items="aperture_method_items.map(i => i.label)"
```

Reply:
(also I think we/glue use title case pretty consistently for choices... so might be worth doing that here too... but we can always kick that decision to the UI/UX review as well)